src/test/java/org/apache/commons/compress/archivers/TarTestCase.java

```diff
@@ -335,6 +482,18 @@ public void testCOMPRESS178() throws Exception {
         in.close();
     }
 
+    @Test
+    public void testTarFileCOMPRESS178() throws Exception {
+        final File input = getFile("COMPRESS-178.tar");
+        try (final TarFile tarFile = new TarFile(input)) {
+            // Compared to the TarArchiveInputStream all entries are read when instantiating the tar file
+            fail("Expected IOException");
+        } catch (final IOException e) {
```

Comment:
Use assertThrows().

Reply:
`assertThrows` is not available in Junit 4. The test is equivalent to the already exisiting test for the `TarArchiveInputStream`. Should I change both tests to use the `expected` property in the `@Test` annotation?

Reply:
I prefer to leave it as is.